### PR TITLE
Add Features "Load prompt from layer"

### PR DESCRIPTION
**Pull request by ElliottLester**
- Add Prompt response class
- More robust failure handling
- Remove all base64 from response path
- Add SSL/HTTPS support for use with comfyanonymous/ComfyUI/pull/1481

### CODE REVIEW

File: frontends/krita/krita_comfy/prompt.py [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+
+_default.default = json.JSONEncoder().default
+json.JSONEncoder.default = _default
+
+
+def get_members(obj):
+    return [attr for attr in dir(obj) if not callable(getattr(obj, attr)) and not attr.startswith("__")]
+
+
+def bind_or_none(data: dict, keys: list[str]):
```
Comment (JasonS09) [inline frontends/krita/krita_comfy/prompt.py]:
```suggestion
def bind_or_none(data: dict, keys: List[str]):
```
Getting an error of type not being suscriptable.

I suggest using the classes from the `typing` module.  You'll need to import `Tuple` as well.
Lines 35, 216, 220 are causing this error.

**Review by JasonS09 — CHANGES_REQUESTED** [review]

File: frontends/krita/krita_comfy/prompt.py (hunk 1 above)
Comment (ElliottLester) [reply frontends/krita/krita_comfy/prompt.py]: Thank you I see
`# On Python 3.8 and earlier, the name of the collection type is`
`# capitalized, and the type is imported from the 'typing' module`

Comment (ElliottLester) [reply frontends/krita/krita_comfy/prompt.py]: As for the tuple looks like variable size is only on 3.9 so I'll stick with a list.
`# For tuples of variable size, we use one type and ellipsis`
`x: tuple[int, ...] = (1, 2, 3)  # Python 3.9+`

**Review by JasonS09 — APPROVED** [review]

**Comment by JasonS09** [comment]
Thanks for your contribution, it's very useful!

**Pull request merged**